modules/rhel-attaching-instance-aws.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="rhel-attaching-instance-aws_{context}"]
+= Attaching the role permissions to {op-system-base} instance in AWS
+
+Using the Amazon IAM console in your browser, you may select the needed roles and assign them to a worker node.
```

Comment:
* Is it really called the "Amazon IAM console", or is it just the regular "AWS console"? Or if IAM console is right, should it be "AWS IAM console"?
* may -> can
* needed -> required